app/views/trainees/lead_schools/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render PageTitle::View.new(title: "pages.search_schools.page_title") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-1"><%= t("pages.search_schools.page_title") %></h1>
+    <div class="govuk-hint">Results for ‘<%= params[:q] %>’</div>
+    <%= register_form_with(model: @trainee_lead_school_form,
+                           url: trainee_lead_schools_path(@trainee),
+                           method: :put,
+                           local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+      <div class="govuk-!-margin-bottom-6">
+        <% @schools.each do |school| %>
+          <%= f.govuk_radio_button :lead_school_id, school.id,
+                                   label: { text: school.name },
+                                   hint: { text: school.urn_and_location } %>
```

Comment:
hmmm.. it _is_ nice to be able to write `school.urn_and_location` though 🤔

Reply:
As you rightly pointed, unless it's used outside of the presentation layer, either we create a decorator object or a helper method. I'm happy to go along with the helper method approach for now.